engine/evd.mli

```diff
@@ -371,12 +371,16 @@ module FutureGoals : sig
                                    a term containing [e]. *)
   }
 
+  type stack
+
   val map_filter : (Evar.t -> Evar.t option) -> t -> t
   (** Applies a function on the future goals *)
 
   val filter : (Evar.t -> bool) -> t -> t
   (** Applies a filter on the future goals *)
 
+  val pr_stack : stack -> Pp.t
```

Comment:
Can't you just export a function taking an evarmap instead of having to export the `stack` type?

Reply:
Done.